lib/job-simulation.rb

```diff
@@ -5,11 +5,34 @@ class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @jobs_available = jobs_available
+    @job_seekers = job_seekers
+    @workers = Stack.new
+    @waiting = Queue.new
+    n = 0
+    @jobs_available.times do
+      @workers.push("Worker ##{n+1}")
+      n += 1
+    end
```

Comment:
Yes - I see why you did it, but it's still strange to me. 

If you don't want it to start with Worker #0, you could say:
```
job_seekers.times do |i|
 @waiting.enqueue("Worker ##{i+jobs_available+1}")
end
```